packages/cli/src/commands/oracle/report.ts

```diff
@@ -0,0 +1,64 @@
+import { CeloContract } from '@celo/contractkit'
+import { flags } from '@oclif/command'
+import BigNumber from 'bignumber.js'
+import { BaseCommand } from '../../base'
+import { displaySendTx } from '../../utils/cli'
+import { Flags } from '../../utils/command'
+
+export default class ReportPrice extends BaseCommand {
+  static description =
+    'Report the price of Celo Gold in a specified token (currently just Celo Dollar, aka: "StableToken")'
+
+  static args = [
+    {
```

Comment:
unsure if we need this option know... since there's only one token.. .maybe make it optional, and use a default value